types/wlr_xdg_foreign_v1.c

```diff
@@ -284,6 +284,7 @@ static void xdg_imported_handle_exported_destroy(struct wl_listener *listener,
 	struct wlr_xdg_imported_v1 *imported =
 		wl_container_of(listener, imported, exported_destroyed);
 	zxdg_imported_v1_send_destroyed(imported->resource);
+	wl_resource_set_user_data(imported->resource, NULL);
```

Comment:
Maybe we should do this from within `finish_imported`, to make sure we never forget to do it?